patches/assimp.patch

```diff
@@ -9,3 +11,31 @@
      "Build a static (.a) version of the library" )
  
  SET(ASSIMP_DEBUG_POSTFIX "d" CACHE STRING "Debug Postfitx for lib, samples and tools")
+diff --git libs/assimp-3.1.1/code/BlenderDNA.cpp libs/assimp-3.1.1/code/BlenderDNA.cpp
+index 16e639e..53e9706 100644
+--- libs/assimp-3.1.1/code/BlenderDNA.cpp
++++ libs/assimp-3.1.1/code/BlenderDNA.cpp
+@@ -55,7 +55,7 @@ using namespace Assimp::Formatter;
+ 
+ #define for_each BOOST_FOREACH
+ bool match4(StreamReaderAny& stream, const char* string) {
+-	char tmp[] = { 
++	int8_t tmp[] = { 

```

Comment:
See comment below.
If possible, apply `static_cast` to each member of the array as below instead of changing the type of the array.